### PR TITLE
Feature: Relative links for pupil search and detail, collection and bundle detail pages

### PR DESCRIPTION
**Pull request by bertyhell**
parts of:
* https://meemoo.atlassian.net/browse/AVO-1910
* https://meemoo.atlassian.net/browse/AVO-1911
* https://meemoo.atlassian.net/browse/AVO-1912

Still TODO:
* Add to my collection button implementation => separate PR

I've also done some work on collections and bundles. which isn't required for pupil search, since they can't see them, but it might still be helpfull in the future to have the same link wrappers for all links in all detail views.
It also reduces some of the duplicated render code between the collection en bundle detail views.

Links inside the search:
image

limit "type" filter to options: video and audio:
image

limit search results to video and audio when searching for collection: "Canon: Wolfijzers en schietgeweren - Richard Minne"
pupil search only returns audio result
image
search search returns both audio and collection result:
image

detail of an item:
image

### CODE REVIEW

File: src/search/components/SearchResultItem.tsx [hunk 1]
```diff
@@ -52,46 +53,58 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
...
-			<SearchResultTitle>
-				<Link to={contentLink}>{result.dc_title}</Link>
-			</SearchResultTitle>
-			{!!result.original_cp && (
-				<SearchResultSubtitle>
-					{generateSearchLink('provider', result.original_cp, 'c-body-2', () =>
-						handleOriginalCpLinkClicked(result.id, result.original_cp || '')
+		<div id={id}>
+			<SearchResult
+				key={`search-result-${result.id}`}
```
Comment (ian-emsens-shd) [inline src/search/components/SearchResultItem.tsx:76]: Lijkt me best om deze waarde ook te gebruiken voor `id` hierboven

File: src/shared/hooks/useCollectionQualityLabels.ts [hunk 2]
```diff
@@ -8,13 +8,16 @@ import { ToastService } from '../services';
 
 type UseCollectionQualityLabelsTuple = [QualityLabel[], boolean];
 
-export const useCollectionQualityLabels = (): UseCollectionQualityLabelsTuple => {
+export const useCollectionQualityLabels = (enabled: boolean): UseCollectionQualityLabelsTuple => {
```
Comment (ian-emsens-shd) [inline src/shared/hooks/useCollectionQualityLabels.ts:11]: Ik zou hier `enabled` default op true zetten of de prop vermijden en de implementaties omvormen naar optional render-functie props.

**Review by ian-emsens-shd — APPROVED** [review]
Enkele kleine opmerkingen, kunnen opgenomen worden in latere PR's

File: src/assignment/views/AssignmentResponseEdit.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+	.m-bundle-detail > .m-bundle-detail-header,
+	.m-collection-detail > .c-header {
+		background-color: transparent;
+	}
+
+	.c-return--search-results {
+		margin: 1rem 0;
+		padding: 0.9rem 0 2.5rem;
+		height: 4rem;
```
[the remark this answers is not in the document]
Comment (bertyhell) [reply src/assignment/views/AssignmentResponseEdit.scss:15]: hoeveel is 1rem? 16px?

**Pull request merged**